src/Helpers/generateToken.js

```diff
@@ -0,0 +1,9 @@
+import jwt from "jsonwebtoken";
+
+// console.log(process.env.JWT_KEY);
+
+export const generateToken = (req, res, next) => {
+  let token = jwt.sign(req.body, "SECRET_KEY");
```

Comment:
Could you store your secret key in `.env` file?

Reply:
I'm having trouble with storing and accessing the env variables that's why I used a string while I'm still trying to figure out the problem.